backend/spec/models/tracking_spec.rb

```diff
@@ -25,9 +25,13 @@
     it { is_expected.to validate_presence_of(:user) }
     it { is_expected.to validate_presence_of(:trackable) }
     it { is_expected.to validate_inclusion_of(:trackable_type).in_array(%w[Condition Symptom Treatment]) }
+
     context "without foreign key checks" do
+      subject { create :tracking, :for_condition, :active }
```

Comment:
Fixed an error during upgrade, this fix's pattern from [shoulda-matchers issue]( https://github.com/thoughtbot/shoulda-matchers/issues/1052).